clue.js

```diff
@@ -0,0 +1,189 @@
+// Suspects objects
+const mrGreen = {
+  firstName: 'Jacob',
+  lastName: 'Green',
+  color: 'green',
+  description: 'He has a lot of connections',
+  age: 45,
+  image: 'assets/green.png',
+  occupation: 'Entrepreneur'
+};
+const proessorPlum = {
+  firstName: 'Victor',
+  lastName: 'Plum',
+  color: 'purple',
+  description: 'He is good at lying',
+  age: 29,
+  image: 'assets/plum.png',
+  occupation: 'Developer'
+};
+const missScarlet = {
+  firstName: 'Cassandra',
+  lastName: 'Scarlet',
+  color: 'red',
+  description: 'She is clever',
+  age: 32,
+  image: 'assets/scarlet.png',
+  occupation: 'Nurse'
+};
+const mrsPeacock = {
+  firstName: 'Elenor',
+  lastName: 'Peacock',
+  color: 'pink',
+  description: 'She is fast',
+  age: 42,
+  image: 'assets/peacock.png',
+  occupation: 'Police officer'
+};
+const colonelMustard = {
+  firstName: 'Jack',
+  lastName: 'Mustard',
+  color: 'grey',
+  description: 'He is funny',
+  age: 56,
+  image: 'assets/mustard.png',
+  occupation: 'Teacher'
+};
+const mrsWhite = {
+  firstName: 'Mrs',
+  lastName: 'White',
+  color: 'white',
+  description: 'She is fast',
+  age: 33,
+  image: 'assets/white.png',
+  occupation: 'chef'
+};
+
+// Objects for weapons
+const rope = {
+  name: 'Rope',
+  weight: 10
+};
+const knife = {
+  name: 'Knife',
+  weight: 100
+};
+const candelstick = {
+  name: 'Candelstick',
+  weight: 200
+};
+const dumbbell = {
+  name: 'Dumbbell',
+  weight: 300
+};
+const poison = {
+  name: 'Poison',
+  weight: 2
+};
+const axe = {
+  name: 'Axe',
+  weight: 400
+};
+const bat = {
+  name: 'Bat',
+  weight: 110
+};
+const trophy = {
+  name: 'Trophy',
+  weight: 500
+};
+const pistol = {
+  name: 'Pistol',
+  weight: 150
+};
+
+// Array of suspects
+const suspects = [
+  mrGreen,
+  proessorPlum,
+  missScarlet,
+  mrsPeacock,
+  colonelMustard,
+  mrsWhite
+]
+suspects.forEach((element) => console.log(element));
+
+//Array of weapons
+const weapons = [
+  rope,
+  knife,
+  candelstick,
+  dumbbell,
+  poison,
+  axe,
+  bat,
+  trophy,
+  pistol
+]
+weapons.forEach((element) => console.log(element));
+
+//Array of rooms
+const rooms = [
+  'Dining rooms',
+  'conservatory',
+  'Kitchen',
+  'Study',
+  'Library',
+  'Billiard room',
+  'Lounge',
+  'Ballroom',
+  'Hall',
+  'Spa',
+  'Living room',
+  'Observatory',
+  'Theater',
+  'Guest house',
+  'Patio'
+]
+
+// Function that randomly selects one item from the array
+const randomSelector = array => {
+  return array[Math.floor(Math.random() * array.length)]
+}
+
+// Object with a killer, a weapon and a room  
+let mystery = {
+  killer: 'unknown',
+  weapon: 'unknown',
+  room: 'none'
```

Comment:
It's far better approach to use `null` values for initialisation :
```
const mystery = {
  killer: null,
  weapon: null,
  room: null
}
```
Moreover, you can use `const` - we will be updating value of this variable, that's true, but since it will still be object, JavaScript will not consider property changes as overwriting value and therefore won't throw error

Reply:
Good point! Updated accordingly above :)